src/test/java/core/basesyntax/service/impl/FruitServiceImplTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitService;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class FruitServiceImplTest {
+    private static final String FRUIT = "apple";
+    private static final int INITIAL_QUANTITY = 10;
+    private static final int SUPPLY_QUANTITY = 60;
+    private static final int NOT_VALID_QUANTITY = 40;
+    private static FruitService fruitService;
+    private static List<FruitTransaction> fruitTransactions;
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        fruitService = new FruitServiceImpl();
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        fruitTransactions = new ArrayList<>();
+        fruitTransactions.add(new FruitTransaction(FruitTransaction.Operation.BALANCE,
+                FRUIT, INITIAL_QUANTITY));
+    }
+
+    @Test
+    public void calculateTotalQuantity_NotEnoughFruits_NotOk() {
```

Comment:
```suggestion
    public void calculateTotalQuantity_notEnoughFruits_notOk() {
```
Look all test methods and rename them according name convention